crates/astria-sequencer-relayer/tests/blackbox/main.rs

```diff
@@ -77,6 +82,8 @@ async fn report_degraded_if_block_fetch_fails() {
         expected: json!({"status": "degraded"}),
         actual: readyz.json::<serde_json::Value>().await.unwrap(),
     );
+
+    sequencer_relayer.shutdown();
```

Comment:
See my comment on replacing `shutdown` with  a `Drop` impl on the mock server so you can get rid of the calls here.